Jenkinsfile

```diff
@@ -1 +1,2 @@
-buildPlugin()
+// Also to test against Jenkins 2.102 with JEP-200.
+buildPlugin(jenkinsVersions: [null, '2.102'])
```

Comment:
Test may fail as I can't specify java.level=8 for 2.102.
Anyway, I'll try this.